ui/js/filters.js

```diff
@@ -33,6 +33,45 @@ treeherder.filter('stripHtml', function() {
     };
 });
 
+treeherder.filter('linkifyURLs', function() {
+    return function(input) {
+        var str = input || '';
+        var urlpattern = /(\b(https?):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gim;
```

Comment:
"Spin your own url regex" is a risky game, but this looks simple enough to catch all of the instances that we actually care about, especially if this filter is only being applied to classification comments.